modules/gltf/gltf_document.cpp

```diff
@@ -4213,6 +4213,35 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> p_state) {
 			}
 		}
 
+		// Supersedes KHR_materials_pbrSpecularGlossiness
+		// see https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md
+		if (material_extensions.has("KHR_materials_specular")) {
```

Comment:
All extensions supported directly by GLTFDocument must be added to `GLTFDocument::get_supported_gltf_extensions_hashset()`.